actions/java-gradle-test/README.md

```diff
@@ -19,6 +19,8 @@ Sonatype, then these steps are skipped.
 | gradle-version          |    ❌    |    wrapper    | string  | [Gradle version](https://github.com/gradle/gradle-build-action#use-a-specific-gradle-version) to be installed |
 | gradle-cache            |    ❌    |     true      | boolean | Whether Gradle caching is enabled or not                                                                      |
 | working-directory       |    ❌    |      "."      | string  | Working directory of your Gradle artifacts                                                                    |
+| download-lfs-files      |    ❌    |     false     | boolean | Whether the Git checkout action should resolve LFS files or not                                               |
```

Comment:
Maybe checkout-lfs-files is better

Reply:
I think technically they are checked out but just to this ["pointer file" format](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage#pointer-file-format). 

In `actions/checkout` they also use "download": https://github.com/actions/checkout/blob/main/README.md?plain=1#L90

Reply:
Ok